index.js

```diff
@@ -0,0 +1,5 @@
+import GitHubLinkExtension from './src/components/GitHubLinkExtension';
+import Extensions from './src/components/Extensions';
+
+export { GitHubLinkExtension };
```

Comment:
nit: should be able to write `export { GitHubLinkExtension , Extensions };`

Reply:
One would think, but that wasn't working.